### ISSUE TITLE
Bootstrap Table extension filter control: Filter control don't work properly when used on multiple tables

### ISSUE THREAD

**Issue by shawnngtq** [wenzhixin/bootstrap-table#2615]
I am using _Bootstrap-table_ & its extension _Filter-control_.

I am building tables using different data sets derived from same data. Hence the different data sets share the same data-field.

When I used Filter-control extension on both tables, the select options will keep increasing with different actions (sort, clear filters, etc).

How can I fix this?

[jsfiddle](https://jsfiddle.net/Shawn_fiddle/hbt8zn2b/)
http://stackoverflow.com/questions/39256750/bootstrap-table-extension-filter-control-filter-control-dont-work-properly-whe

**Comment by dabros** [comment]
@wenzhixin @Shawn-2016 - both fiddle and stackoverflow lead to 404 pages, issue should be closed or updated

**Comment by mwiest** [comment]
I can confirm this on my end, having two similar tables on the same page, both with filters and the select values are duplicating like crazy. Even the ones explicitly defined by data-filter-data="".
I can't give you a Fiddle, but should be easy to reproduce.

**Comment by Falseee** [comment]
Can not reproduce the issue in the newest version.

**Comment by langert** [comment]
In V 12.4 I got an similar problem. Haveing three tables with different data and different table ids. Initialized by javascript. All filtarable columns, which have a name, existing in one of the other tables, are broken. The select-filter does not show any values. Changing from select to input , I can start a search but the query url for all three tables is the same - the one from the last initialized table. 
Tried to update to last V 1.22. but I can not get running this version with my webpack configuration :-(
When I delete tow of the three table. Each table for its own is running fine. 
Any hints on solving the multiple table problem?
